iacrcc/iacrcc.cls

```diff
@@ -459,34 +460,26 @@ xscale=\@OrigHeightRecip*\@curXheight,transform shape]
   }%
 }
 
-% Add support to display provided e-mail addresses 
+% Add support to display the provided e-mail addresses 
 % in the footnote of the front page
 \global\let\IACR@displayemails\@empty
 
 % Add support for a footnote without symbol on the front page
 \global\let\@genericfootnote\@empty
-\newcommand\genericfootnote[1]{
+\newcommand\genericfootnote[1]{%
   \gdef\@genericfootnote{#1}%
 }
 
 \def\@maketitle{%
-  % Count authors and affiliations
-  \setcounter{IACR@author@cnt}{1}%
-  \setcounter{IACR@inst@cnt}{1}%
-  \setbox0\hbox{\def\thanks##1{\global\@IACR@autoinstfalse}\def\inst##1{\global\@IACR@autoinstfalse}\def\and{\stepcounter{IACR@author@cnt}}\@author}%
-  \setbox0\hbox{\def\and{\stepcounter{IACR@inst@cnt}}\@affiliation}%
-  \xdef\IACR@author@last{\theIACR@author@cnt}%
-  \edef\IACR@inst@last{\theIACR@inst@cnt}%
-  \ifnum\IACR@author@last=\IACR@inst@last\else\@IACR@autoinstfalse\fi
-  \ifnum\IACR@author@last=1 \@IACR@autoinstfalse\fi
```

Comment:
what in the world was all this stuff? Is this for the case when there is only one affiliation so we don't need footnotes?

Reply:
This was counting the number of authors (which was much more difficult when \author was used instead of \addauthor) and attempting to automatically inserting \inst markers (when needed).